### DISCUSSION TITLE
How to unfold all implementations in vscode?

### DISCUSSION THREAD

**Discussion by xzbdmw** [rust-lang/rust-analyzer#15740] [Q&A]
This is annoying to manually unfold each file to find the corresponding symbol, is there any good ways to do it?
<img width="1610" alt="image" src="https://github.com/rust-lang/rust-analyzer/assets/97848247/85bb9f28-5fc9-4ead-84df-22f36af15a23">

**Comment by Veykril** [comment]
Not to my knowledge, that's a VSCode UX issue so you'd need to ask there.
